receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -196,3 +196,22 @@ query getPullRequestData(
     }
   }
 }
+
+query getRepoCVEs($owner: String!, $repo: String!) {
+  repository(owner: $owner, name: $repo, ) {
+    vulnerabilityAlerts(first: 100, states: OPEN) {
+      totalCount
```

Comment:
Do we need this? We don't get the total count. We get the count by severity & then create total count on the visualization side. This is a pretty non-impactful query, but was debating whether or not we needed it.

Reply:
We're not using it now, so no. I removed it.